python/paddle/fluid/memory_optimization_transpiler.py

```diff
@@ -118,7 +118,7 @@ def _find_var(self, block_desc, var_name, is_forward):
         else:
             return block_desc.find_var_recursive(str(var_name))
 
-    def memory_optimize(self):
+    def memory_optimize(self, level=0):
```

Comment:
The code style says that a function name should be a verb-subject phrase, like, optimize_memory, instead of memory_optimize.

Also, it seems that we cannot optimize the memory; what we could is to optimize the usage of the memory.

For this case, does it mean `reuse_memory` and should we rename `level` into `reuse_tensor_with_the_same_size`?

Reply:
Yes, you are right. It's mainly reuse memory. And level 0 means that we can reuse tensor with the same size. Level 1 means that we can reuse tensor if current tensor size is the same or less than cache pool tensor.

I will refine these codes accordingly. Thanks!